erts/doc/src/erlang.xml

```diff
@@ -9053,6 +9053,18 @@ ok
               <seealso marker="doc/system_principles:versions">
               System principles</seealso> in System Documentation.</p>
           </item>
+          <tag><marker id="system_info_otp_minor_version"/>
+            <c>otp_minor_version</c></tag>
+          <item>
+            <marker id="system_info_otp_minor_version"></marker>
+            <p>Returns a string containing the OTP minor version number of the
+              OTP release that the currently executing ERTS application
+              is part of. As an example if the OTP version is 21.2.3, 
+              this would return "2".</p>
```

Comment:
Below, you state "... as an integer", but here it gives the aspect of a string ("2").

Reply:
Will fix up.

Reply:
@paulo-ferraz-oliveira  I think there is some confusion. The macro returns an integer, system_info/1 does not not unlike ?OTP_RELEASE and `system_info(otp_release)`